src/KafkaFlow.Sample/appsettings.Development.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
@eduardosbcabral , como você pretende implementar as variáveis de ambiente na app que for usar a biblioteca?

Seria usando o arquivo appsettings mesmo e fazendo replace de acordo com o ambiente?

Reply:
Sim, eu prefiro appSettings pq mesmo que não precise na hora de deployar é melhor para entender o nome das variáveis que a aplicação vai usar, em vez de deixar somente no chart.

Reply:
Mas vocês deixam no appsettings para o ambiente de desenvolvimento e os charts para staging e produção, @eduardosbcabral ?

Reply:
@elnatantorres Sim, dev usamos o arquivo e o resto variáveis de ambiente